python-sdk/tutorials/automl-with-azureml/forecasting-orange-juice-sales/auto-ml-forecasting-orange-juice-sales.ipynb

```diff
@@ -403,7 +403,8 @@
     "|**training_data**|Input dataset, containing both features and label column.|\n",
     "|**label_column_name**|The name of the label column.|\n",
     "|**compute_target**|The remote compute for training.|\n",
-    "|**n_cross_validations**|Number of cross-validation folds to use for model/pipeline selection|\n",
+    "|**n_cross_validations**|Number of cross-validation folds to use for model/pipeline selection. Could be customized as an integer. If users don't set it or set it to \"auto\", AutoMl will determine the integer automatically for you.|\n",
```

Comment:
Maybe we can say:
Could be set to custom integer value. If **it is not set or set** it to \"auto\", AutoMl will determine the number of cross validations automatically.

Do we interpret n_cross_validations=None as "auto"?

Reply:
If users leave n_cross_validations not set, and no validation data was provided, we default n_cross_vaidations as "auto".